Knoema.Client/Series/TimeSeriesRequestBuilder.cs

```diff
@@ -81,14 +81,14 @@ public IEnumerable<string> Frequencies
 						_dimensions.Remove(dimId);
 					else
 					{
-						Dictionary<string, int> keys;
+						Dictionary<string, string> keys;
 						if (!_dimensions.TryGetValue(dimId, out keys))
 						{
-							keys = new Dictionary<string, int>();
+							keys = new Dictionary<string, string>();
 							_dimensions[dimId] = keys;
 						}
 						foreach (var key in value)
-							keys[key] = 0;
+							keys[key] = "";
```

Comment:
string.Empty

Reply:
@BillSytin then there should be instead null?